reconnecting-eventsource.d.ts

```diff
@@ -0,0 +1,21 @@
+export interface Configuration extends EventSourceInit {
```

Comment:
I would suggest this name, to match the main object called `ReconnectingEventSource`.

```suggestion
export interface ReconnectingEventSourceInit extends EventSourceInit {
```

Reply:
changed